packages/cli/src/cli.ts

```diff
@@ -53,14 +53,33 @@ export async function cli(args: string[] = process.argv) {
           .demandCommand();
       },
       handler: (_args) => {},
+    })
+    .fail(async (msg, err, yargs) => {
+      const Consola = await import("consola");
+      if (err instanceof ExitProcessError) {
```

Comment:
There are some places that use ExitProcessError with `errorCode` which we don't log anywhere but note that it's not an exit code but the error code from the http requests. This means we're currently logging the stacktrace by using the whole `err` object. I could see an option where we just always include the stacktrace in verbose mode as well